lib/dotcom_web/components/system_status/subway_status.ex

```diff
@@ -131,7 +131,7 @@ defmodule DotcomWeb.Components.SystemStatus.SubwayStatus do
     @route_ids
     |> Enum.map(&{&1, subway_status |> Map.get(&1)})
     |> Enum.flat_map(&rows_for_route/1)
-    |> maybe_collapse_rows()
+    |> collapse_rows_if_needed()
```

Comment:
I guess I could name this back to `maybe_collapse_rows()`. I can't remember now why I changed it.